### PR TITLE
Compose

### PR DESCRIPTION
**Pull request by chuckbutler**
Adds support for installing docker compose to the base charm delivery pattern.

This offers up interesting paths to launching containers and container configuration from docker-compose.yaml files. (consumeable in actions, charms, etc.)

### CODE REVIEW

**Comment by mbruzek** [comment]
+1 LGTM

File: hooks/setup.sh [hunk 1]
```diff
@@ -4,4 +4,6 @@ set -e
 apt-get install -y \
     python-dev \
     python-pip \
-    git \
+    git
+
+easy_install -U pip
```
Comment (mbruzek) [inline hooks/setup.sh]: Do we need both python-pip and eazy_install pip?

Comment (whitmo) [inline hooks/setup.sh]: it's a workaround due to a bug in the pip pkg.

**Comment by whitmo** [comment]
:+1:

**Pull request merged**